src/hooks/use-fetch-wpcom-sites.tsx

```diff
@@ -119,6 +118,14 @@ export const useFetchWpComSites = ( connectedSites: SyncSite[] ) => {
 			} );
 	}, [ client?.req, connectedSites, isAuthenticated, isOffline ] );
 
+	useEffect( () => {
+		fetchSites();
+	}, [ fetchSites ] );
+
+	const refetchSites = useCallback( () => {
+		fetchSites();
+	}, [ fetchSites ] );
```

Comment:
@nightnei , Maybe we could remove this extra function. I don't see a difference between `fetchSites` and `refetchSites`.